hasher-matcher-actioner/hmalib/common/config.py

```diff
@@ -233,7 +233,10 @@ def _aws_field_to_py(in_type: t.Type[T], aws_field: t.Any) -> T:
     # Another option would be adding a new class that adds methods to convert
     # to an AWS-friendly struct and back
     if origin is dict and args[0] is str:  # M
-        return {k: _aws_field_to_py(args[1], v) for k, v in aws_field.items()}  # type: ignore # mypy/issues/10003
+        if args[1] is not t.Any:
```

Comment:
Hmm, This unfortunately breaks typing guarantees that the things you are putting in these maps are serializable. (The purpose of the exception on 241). 

I am aware of a solution that I think solves both of our problems, let me sync with you on next steps.